aptos-move/aptos-replay-benchmark/src/runner.rs

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::block::Block;
+use aptos_vm::{aptos_vm::AptosVMBlockExecutor, VMBlockExecutor};
+use std::time::Instant;
+
+/// Holds configuration for running the benchmarks and measuring the time taken.
+pub struct BenchmarkRunner {
+    concurrency_levels: Vec<usize>,
+    num_repeats: usize,
+    measure_block_time: bool,
```

Comment:
nit: `measure_block_times`. Also it's not clear what's the alternative. so maybe rename to `measure_per_block_instad_of_overall_time`. Or introduce an enum?

Reply:
Renamed